debezium-core/src/main/java/io/debezium/util/Metronome.java

```diff
@@ -50,8 +50,12 @@ public static Metronome sleeper(long period, TimeUnit unit, Clock timeSystem) {
 
             @Override
             public void pause() throws InterruptedException {
-                while (next > timeSystem.currentTimeInMillis()) {
-                        Thread.sleep(next - timeSystem.currentTimeInMillis());
```

Comment:
Couldn't we simply do `Thread.sleep(abs(next - timeSystem.currentTimeInMillis())`?